components/brave_wallet/common/eth_sign_typed_data_helper.cc

```diff
@@ -79,7 +79,9 @@ std::string EthSignTypedDataHelper::EncodeType(
   for (size_t i = 0; i < type.GetList().size(); ++i) {
     const std::string* type_str = type.GetList()[i].FindStringKey("type");
     const std::string* name_str = type.GetList()[i].FindStringKey("name");
-    DCHECK(type_str && name_str);
+    if (!type_str || !name_str) {
```

Comment:
Should I drop the `{ }` here? Seems like it's not needed, but I believe the linter has been bringing this back in

Reply:
Please keep it because linter requires it